File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        leftPointer = 0
        rightPointer = len(nums)
        '''
        [2,3,4]
        if curr element is val:
            What do we do then?
        '''
        while(val in nums):
            nums.remove(val)
        
        return len(nums)
        

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[2*nums.length];
        int SavedIndex = 0;
        for( int i = 0; i < nums.length; i++){
            ans[i] = nums[i];
            SavedIndex++;
        }
        for(int i = SavedIndex ; i < ans.length; i++){
            ans[i] = nums[i - nums.length];   
        }
        return ans;
    }
        
}